functions/_tide_item_ruby.fish

```diff
@@ -0,0 +1,6 @@
+function _tide_item_ruby
+    if path is $_tide_parent_dirs/{*.gemspec,Gemfile,Rakefile,.ruby-version,.tool-versions}
```

Comment:
Might this be better to be something like

```fish
if command --query ruby && path is …
```

This would not help if we were using jruby or another version with a prefix, but it would prevent cases where ruby is selected but not yet installed.

Reply:
That should be taken care of by this change in `_tide_remove_unusable_items`: 

https://github.com/IlanCosman/tide/pull/437/files#diff-fc249ac3efaa608703553322a4c2d675da3f1e4bd3456e861cdd638ca571409cR4

We check for `ruby` on startup.